lib/tds/protocol.ex

```diff
@@ -1175,4 +1175,24 @@ defmodule Tds.Protocol do
         )
     end
   end
+
+  defp append_opts(conn, opts, :set_cursor_close_on_commit) do
+    case Keyword.get(opts, :set_cursor_close_on_commit) do
+      val when val in [nil, :off] ->
```

Comment:
Can we create another case only for `nil`? I would rather leave this to default database settings if this flag is not set at all in `config/config.exs`. Also, in README.md we could add a hint to data warehouse user that this should be set explicitly in connection configuration.